linebot/client.go

```diff
@@ -65,6 +65,7 @@ const (
 	APIEndpointLinkToken = "/v2/bot/user/%s/linkToken"
 
 	APIEndpointGetMessageDelivery = "/v2/bot/message/delivery/%s"
+	APIEndpointInsight            = "/v2/bot/message/insight/%s"
```

Comment:
seems `/insight` is valid, not `/message/insight`
https://developers.line.biz/en/reference/messaging-api/#get-number-of-delivery-messages